test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -1,16 +1,28 @@
 // <reference types="Cypress" />
 describe('Preview Document Thumbnails', () => {
     const token = Cypress.env('ACCESS_TOKEN');
-    const fileId = Cypress.env('FILE_ID_DOC');
+    const fileId = Cypress.env('FILE_ID_DOC_LARGE');
     const THUMBNAIL_SELECTED_CLASS = 'bp-thumbnail-is-selected';
 
     /* eslint-disable */
-    const getThumbnail = (pageNum) => cy.get(`.bp-thumbnail[data-bp-page-num=${pageNum}]`);
+    const getThumbnail = (pageNum) => {
+        return cy
+            .get(`.bp-thumbnail[data-bp-page-num=${pageNum}]`, { timeout: 5000 })
+            .as('foundThumbnail')
+    };
+
+    const getThumbnailWithRenderedImage = (pageNum) => {
+        getThumbnail(pageNum).find('.bp-thumbnail-image').should('exist');
+
+        return cy.get('@foundThumbnail');
```

Comment:
Sharing aliases across helpers feel a bit awkward. Is the `.should` assertion needed everywhere that we want to get the thumbnail? I believe any call to `.find` or `.get` carries an implicit `.should('exist')` automatically.